daemon/src/routes_taker.rs

```diff
@@ -96,6 +96,27 @@ pub async fn post_order_request(
         .expect("actor to always be available");
 }
 
+#[rocket::post("/cfd/<id>/<action>")]
+pub async fn post_cfd_action(
+    id: OrderId,
+    action: CfdAction,
+    cfd_actor_address: &State<Address<taker_cfd::Actor>>,
+) -> Result<status::Accepted<()>, status::BadRequest<String>> {
+    match action {
+        CfdAction::Accept | CfdAction::Reject => {
+            unreachable!("The taker does not accept and reject");
+        }
+        CfdAction::Commit => {
+            cfd_actor_address
+                .do_send_async(taker_cfd::Commit { order_id: id })
+                .await
+                .expect("actor to always be available");
```

Comment:
We should actually also not panic here but return a 500. No network API ever should make it possible to panic an internal process as that can be a security issue.

Reply:
I agree, should fix this in the other API calls at some point, we have several expects there. I only fixed it in the ones concerned by this PR for now, to not cause potential unwanted side effects.